### PR TITLE
Climber Automation

### PR DESCRIPTION
**Pull request by JoeJanickiChSh**
Still requires testing.

### CODE REVIEW

File: .Glass/glass.json [hunk 1]
```diff
@@ -87,15 +106,34 @@
     "Plot <0>": {
       "plots": [
         {
+<<<<<<< HEAD
```
Comment (msoucy) [inline .Glass/glass.json]: 🚩 🚩 🚩

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -180,6 +190,27 @@ public void configureButtonBindings() {
...
+                        leftClimber.autoClimb(),
+                        rightClimber.autoClimb()));
+
+        copilotController.rbumper().whileHeld(
+
+                parallel("Climb Individual",
+
+                        leftClimber.climb(
+                                deadbandAxis(0.15, () -> -copilotController.getLeftY()),
+                                deadbandAxis(0.15, () -> 0)),
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: Why call `deadbandAxis` if it's just going to be 0?

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -180,6 +190,27 @@ public void configureButtonBindings() {
...
+
+                        rightClimber.climb(
+                                deadbandAxis(0.15, () -> -copilotController.getRightY()),
+                                deadbandAxis(0.15, () -> 0))
```
Comment (msoucy) [inline src/main/java/frc/robot/Robot.java]: Why call `deadbandAxis` if it's going to be 0?

File: src/main/java/frc/robot/subsystems/Drive.java [hunk 4]
```diff
@@ -95,6 +95,10 @@ public CommandBase setSpeedCoef(double fac) {
         });
     }
 
+    public double getPitch() {
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Drive.java]: Why's this here?

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 5]
```diff
@@ -72,7 +76,7 @@ public Command testSpeed(double speed) {
 
     public CommandBase setTargetAndStartShooter(HubSpeed hub) {
         return instant("Set Default Speed", () -> {
-            shootSpeed = hub.get();
+            shootSpeed = highGoal;
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Shooter.java:90]: Why even pass in the hub parameter if you're not going to use it?

**Review by msoucy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 5 above)
Comment (JoeJanickiChSh) [reply src/main/java/frc/robot/subsystems/Shooter.java:90]: The `highGoal` variable is only used to dynamically change it through the SmartDashboard. After we find the proper speed for the high goal, the command will use the `HubSpeed` enum.

**Pull request closed without merging**